default.nix

```diff
@@ -21,7 +21,11 @@
 # Once the build is finished, you will find, in the current directory,
 # a symlink to where Coq was installed.
 
-{ pkgs ? (import <nixpkgs> {})
+{ pkgs ?
```

Comment:
Why this change? Is that fetching a concrete Nix image?

Reply:
I guess we discussed that already; and we were in favor of not pinning within `default.nix` (which is used daily by developers). If you really need pinning, please do so in a separate file that has to be explicitly called; in the CI script, you may call `nix-build -f reproducible.nix …`, but when working as usual, I’ll only run plain `nix-shell`.